tutorials/oauth2-with-23andMe-in-java/index.md

```diff
@@ -0,0 +1,17 @@
+This sample app uses Google's OAuth 2.0 java lib to authenticate with 23andMe.  Once authenticated the app pulls a single genotype for i3003137, a SNP associated with Sickle Cell Anemia.
+ 
+### Instructions
+* Create a 23andMe client at https://api.23andme.com/apply/ 
```

Comment:
Might be write this down as normal text instead of bullet-list items. It just makes markdown harder to write and users follow the tutorials linearly anyway. You can have it like:

```
### Step 1: do this

Explain

    command

### Step 2: do this

Explain

    command

```